plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -153,6 +164,11 @@ public DistributedQueryRunner build()
                 queryRunner.createCatalog(ICEBERG_CATALOG, "iceberg", icebergProperties.buildOrThrow());
                 schemaInitializer.orElseGet(() -> SchemaInitializer.builder().build()).accept(queryRunner);
 
+                if (catalog2IcebergProperties.isPresent()) {
```

Comment:
Move this logic to `TestS3WithDoubleSlashPathAndS3ASchemeTest` because it is used only there at the moment.